adofai-roulette.py

```diff
@@ -2,74 +2,50 @@
 
 # setup song, difficulty, and creator lists
 
-tempfile = open("songs.txt", encoding="utf-8")
-songlistbad = tempfile.readlines()
-tempfile2 = open("difficulties.txt", encoding="utf-8")
-difficultylistbad = tempfile2.readlines()
-tempfile3 = open("creators.txt", encoding = "utf-8")
-creatorlistbad = tempfile3.readlines()
+def readLinesAndStrip(filename: str) -> list[str]:
+    with open(filename, encoding="utf-8") as file:
+        lines = file.readlines()

```

Comment:
Note the `with` here, this is known as a context manager. For files, always use a context manager, as opposed to manually calling `file.open()` then `file.close()`. I'd look up more info on this if you're curious.